plugins/search-backend-node/src/test-utils/TestPipeline.ts

```diff
@@ -79,6 +111,48 @@ export class TestPipeline {
     );
   }
 
+  /**
+   * Create a test pipeline given a collator you want to test.
+   */
+  static fromCollator(collator: Readable) {
+    return new TestPipeline({ collator });
+  }
+
+  /**
+   * Add a collator to the test pipeline.
+   */
+  withCollator(collator: Readable) {
```

Comment:
```suggestion
  withCollator(collator: Readable): this {
```

and then returning `this` of course, in all of these `withX` methods.